sugardough/sugardough/base/helpers.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib.staticfiles.templatetags.staticfiles import static as static_helper
```

Comment:
Why the `as` alias? Does it not work to just leave it as `static` and then further down do `static = register.function(static)`?